odoo/fields.py

```diff
@@ -2800,6 +2802,122 @@ def _record_ids_per_res_model(self, records):
         return model_ids
 
 
+class Command(enum.IntEnum):
+    """
+    :class:`~odoo.fields.One2many` and :class:`~odoo.fields.Many2many` fields
+    expect a special command to manipulate the relation they implement.
+
+    Internally, each command is a 3-elements tuple where the first element is a
+    mandatory integer that identifies the command, the second element is either
+    the related record id to apply the command on (commands update, delete,
+    unlink and link) either 0 (commands create, clear and set), the third
+    element is either the ``values`` to write on the record (commands create
+    and update) either the new ``ids`` list of related records (command set),
+    either 0 (commands delete, unlink, link, and clear).
+
+    Via Python, we encourage developers craft new commands via the various
+    functions of this namespace. We also encourage developers to use the
+    command identifier constant names when comparing the 1st element of
+    existing commands.
+
+    Via RPC, it is impossible nor to use the functions nor the command constant
+    names. It is required to instead write the literal 3-elements tuple where
+    the first element is the integer identifier of the command.
+    """
+
+    CREATE = 0
+    UPDATE = 1
+    DELETE = 2
+    UNLINK = 3
+    LINK = 4
+    CLEAR = 5
+    SET = 6
+
+    @classmethod
+    def create(cls, values: dict):
+        """
+        Create new records in the comodel using ``values``, link the created
+        records to ``self``.
+
+        In case of a **Many2many** relation, one unique new record is created
+        in the comodel such that all records in `self` are linked to the new
+        record.
+
+        In case of a **One2many** relation, one new record is created in the
+        comodel for every record in ``self`` such that every record in ``self``
+        is linked to exactly one of the new records.
+
+        Return the command triple (CREATE, 0, ``values``)
```

Comment:
These should probably use `:samp:`, as-is I think they're going to look weird e.g.

> Return the command triple (CREATE, 0, `values`)

versus

> Return the command triple `(CREATE, 0, *values*)`

(in actual rendering `values` here would be emphasized within a monospaced block)

The latter makes the "technicality" of the triplet and the identity of `values` clearer.